foundry/foundry.py

```diff
@@ -396,6 +396,28 @@ def publish(self, foundry_metadata, data_source, title, authors, update=False,
         res = self.connect_client.submit_dataset(update=update)
         return res
 
+    def publish_model(self, title, creators, short_name, servable_type, serv_options):
+        """Simplified publishing method for servables
+
+        Args:
+            title (string): title for the servable
+            creators (string | list): either the creator's name (FamilyName, GivenName) or a list of the creators' names
+            short_name (string): shorthand name for the servable
+            servable_type (string): the type of the servable, must be a member of ("static_method",
+                                                                                   "class_method",
+                                                                                   "keras",
+                                                                                   "pytorch",
+                                                                                   "tensorflow",
+                                                                                   "sklearn")
+            serv_options (dict): the servable_type specific arguments that are necessary for publishing
```

Comment:
please link to where the user may find the list of arguments, and how they're applied

Reply:
let me know if the new changes leave anything to be desired

Reply:
look good!